.github/workflows/deployment-template.yml

```diff
@@ -77,4 +134,31 @@ jobs:
             helm uninstall dashboards --namespace default
             kubectl get pvc | grep opensearch-cluster-leader-opensearch-cluster-leader- | awk '{print $1}'| xargs kubectl delete pvc
             helm install opensearch opensearch/opensearch -f config/playground/helm/${{inputs.deploy-env}}/helm-opensearch.yaml
-            helm install dashboards opensearch/opensearch-dashboards -f config/playground/helm/${{inputs.deploy-env}}/helm-opensearch-dashboards.yaml
\ No newline at end of file
+            helm install dashboards opensearch/opensearch-dashboards -f config/playground/helm/${{inputs.deploy-env}}/helm-opensearch-dashboards.yaml
+
+      - name: Step 8 - Install Logstash
+        if: ${{inputs.deploy-env == 'preview' }}
+        uses: elastic-analytics/dashboards-action@main
```

Comment:
looks like step 5-9 all uses this `elastic-analytics/dashboards-action@main`, the config is the same and they all have the condition check `if: ${{inputs.deploy-env == 'preview' }}`, why do we want to separate them?

Reply:
step 5-9's components were not existing in other environment, which we need to check if env is correct, it is not related with action self.